### PR TITLE
Remove the support for updates on a floppy disk

### PR DESCRIPTION
**Pull request by poncovka**
This is the last floppy-related code that we have in our code base.

### CODE REVIEW

**Review by jkonecny12 — APPROVED** [review]
Looks good to me from the code PoV.
I wonder, are we sure that floppy disks are not used anymore. I mean some outpost somewhere in the middle of nowhere may still use these.

File: dracut/parse-kickstart [hunk 1]
```diff
@@ -163,9 +163,7 @@ class DracutURL(Url, DracutArgsMixin):
 
 class DracutUpdates(Updates, DracutArgsMixin):
     def dracut_args(self, args, lineno, obj):
-        if self.url == "floppy":
-            return "live.updates=/dev/fd0"
-        elif self.url:
+        if self.url:
```
Comment (VladimirSlavik) [inline dracut/parse-kickstart:166]: What happens if "floppy" is there? Intentional error?

Comment (poncovka) [reply dracut/parse-kickstart:166]: There will be no officially supported way how to set `url` to `floppy`. Users were supposed to specify `updates` without any arguments to use the floppy disk and that won't be possible with the pykickstart update. It is possible to specify `updates floppy` in a kickstart file, but the documentation says that the argument should be a URL.

Comment (VladimirSlavik) [reply dracut/parse-kickstart:166]: OK, changing to ACK then.

I understand this as "a device can be there too but we are not telling users that". Which is fine.

**Review by VladimirSlavik — APPROVED** [review]

**Comment by poncovka** [comment]
> I wonder, are we sure that floppy disks are not used anymore. I mean some outpost somewhere in the middle of nowhere may still use these.

I guess that users can still use `/dev/fd0`, so we might want to keep the kernel module loaded. Btw, the `updates` kickstart command is not documented at all in RHEL 8.

**Comment by poncovka** [comment]
Pykickstart is fixed in: https://github.com/pykickstart/pykickstart/pull/361

**Comment by jkonecny12** [comment]
> > I wonder, are we sure that floppy disks are not used anymore. I mean some outpost somewhere in the middle of nowhere may still use these.
> 
> I guess that users can still use `/dev/fd0`, so we might want to keep the kernel module loaded. Btw, the `updates` kickstart command is not documented at all in RHEL 8.

That may be on purpose. It's really just for debugging, nothing what users should use on daily basis.

**Comment by poncovka** [comment]
The kernel module remains loaded.

**Review by jkonecny12 — APPROVED** [review]
Looks great to me now!

**Pull request merged**